### ISSUE TITLE
Some question about Reddit dataset

### ISSUE THREAD

**Issue by wjlpku** [GraphSAINT/GraphSAINT#4]
Hi, congratulations to you, it seems like a good score in ICLR open-review.
Here I have some questions about reddit dataset, when I print the "role.json" of reddit, 
I find the train/val/test: 151701/23699/55334, however in cluster-gcn paper, 
it reports the train/val/test: 153932/23699/55334.
I wonder how the difference between this two paper in split size on reddit?

**Comment by GraphSAINT** [comment]
Thanks for letting us know. The split should be the same as ClusterGCN (153932/23699/55334). For some reason, the uploaded `role.json` was corrupted. 

We will upload the correct version within one day, and let you know. The performance numbers reported in the paper are not affected.

**Comment by GraphSAINT** [comment]
We have concluded our investigation. 

**Cause of the problem:**

Since Reddit is originated from GraphSAGE, so we generate the Reddit adj in the format of GraphSAINT by running the public GraphSAGE code. However, when GraphSAGE loads Reddit into Networkx graph, the networkx conversion automatically drops a small number of nodes (for a reason currently unclear to us). So when we further convert the networkx graph into a scipy sparse adjacency matrix, it appears that the split became 151701/23699/55334. In summary, we didn't do any additional pre-processing on Reddit to alter the split, it is simply a result of some internal processing of the library.

**Solution:**

To avoid any confusion, we have re-generated the Reddit graph in the GraphSAINT format. The re-generated one is exactly the same as the one used by ClusterGCN, with the split 153932/23699/55334. We have uploaded the new Reddit graph. In the Google drive link provided in `README`, you will find the new graph under the same `reddit/` directory. 

**Conclusion:**

All the configuration files and the corresponding accuracy remain unchanged. We have also double-checked to ensure the consistency of the other datasets.
